src/test/java/codeu/controller/AdminServletTest.java

```diff
@@ -0,0 +1,51 @@
+// Copyright 2017 Google Inc.
```

Comment:
Why is the whole file commented out? :D
Also, since the AdminServlet does some logic, asserting that the requestDispatcher is being forwarded is not sufficient to fully cover the servlet functionality. Maybe you wanna test if it will call `sendRedirect` when the username is not "admin", and test the forward when the username is indeed "admin". Those will be two separate test methods in the same test class